iOSLearningApp/DetailView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DetailView.swift
+//  iOSLearningApp
+//
+//  Created by Chioma Amanda Mmegwa  on 09/10/2022.
+//
+
+import SwiftUI
+
+struct DetailView: View {
+    var body: some View {
+        VStack {
+            header
+            newsDetail
+        }
+        .padding()
+        .ignoresSafeArea()
+        .navigationBarHidden(true)
+    }
+    var header: some View {
+        NavigationBar(navigationTitle: "DETAIL", rightIcon1: "square.and.arrow.up", rightIcon2: "bookmark")
```

Comment:
please create a constant file for the string to make localisation easier

Reply:
check out this library https://github.com/mac-cain13/R.swift/ it makes import of assets easier